testproject/tests/test_second_step_authentication.py

```diff
@@ -809,39 +763,87 @@ def test_backup_codes_regeneration_disabled_method(
         )
     )
     response = client.post(
-        path="/auth/sms/codes/regenerate/",
+        path="/auth/sms_twilio/codes/regenerate/",
+        data={
+            "code": handler.create_code(),
+        },
         format="json",
     )
-    error_msg = "Method is disabled."
     assert response.status_code == 400
```

Comment:
in other tests we are using `rest_framework.status` const so we can do in this test too